src/http/cf.rs

```diff
@@ -0,0 +1,89 @@
+use std::time::Duration;
+
+use cloudflare::framework::auth::Credentials;
+use cloudflare::framework::response::ApiFailure;
+use cloudflare::framework::{Environment, HttpApiClient, HttpApiClientConfig};
+use http::StatusCode;
+
+use crate::http::{feature::headers, Feature};
+use crate::settings::global_user::GlobalUser;
+use crate::terminal::{emoji, message};
+
+pub fn auth_client(user: &GlobalUser) -> Result<HttpApiClient, failure::Error> {
+    let config = HttpApiClientConfig {
+        http_timeout: Duration::from_secs(30),
+        default_headers: headers(None),
+    };
+
+    HttpApiClient::new(
+        Credentials::from(user.to_owned()),
+        config,
+        Environment::Production,
+    )
+}
+
+pub fn kv_auth_client(user: &GlobalUser) -> Result<HttpApiClient, failure::Error> {
```

Comment:
maybe we rename to `long_timeout_auth_client`?

Reply:
yeah prolly. or we handle config less centrally in another iteration of this refactor